packages/core/introspect-mcp/README.md

```
@@ -0,0 +1,108 @@
+### 2. Interactive probe with MCP Inspector
+
+The MCP SDK ships an interactive UI for talking to a stdio server. From the monorepo root:
+
+```bash
+npx @modelcontextprotocol/inspector \
+  npx tsx packages/core/introspect-mcp/src/cli.ts --root "$PWD"
+```
+
+> ⚠️ **Don't launch via `pnpm start`.** pnpm's script-runner writes a header line (`> @dxos/introspect-mcp@x.y.z start`) to stdout, which corrupts the JSON-RPC stream and produces "Connection Error – Check if your MCP server is running and proxy token is correct." in the Inspector. 
+  Invoke `tsx` (or the built binary) directly so the server owns stdout.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)